publish/publish_workflow.wdl

```diff
@@ -0,0 +1,667 @@
+        | sed '$d' \
+        > changelog.txt
+
+        echo 'Build the json body for the POST release'
+        jq \
+            --null-input --rawfile changelog changelog.txt '{
+                tag_name: "~{currentReleaseVersion}",
+                name: "~{currentReleaseVersion}",
+                body: $changelog,
+                draft: true,
+                prerelease: false
+            }' \
+        > changelog.json
+
+        echo 'POST the release as a draft'
+        curl \
+            --fail --silent \
+            --header "Authorization: token $(cat ~{githubTokenFile})" \
+            --request POST \
+            --data @changelog.json \
+            --output releaseResponse.json \
+            https://api.github.com/repos/~{organization}/cromwell/releases
+
+        echo 'parse the response to get the release id and the asset upload url'
+        jq --raw-output --exit-status .id < releaseResponse.json > releaseId.txt
+        jq --raw-output --exit-status .upload_url < releaseResponse.json > uploadUrl.txt
+    >>>
+
+    output {
+        String releaseId = read_string("releaseId.txt")
+        String uploadUrl = sub(read_string("uploadUrl.txt"), "\\{.*", "")
+        Boolean done = true
+    }
+
+    runtime {
+        docker: publishDocker
+    }
+}
+
+task publishGithubRelease {
+    input {
+        File githubTokenFile
+        String organization
+        String releaseVersion
+        String releaseId
+        String uploadUrl
+        File cromwellJar
+        File womtoolJar
+        String publishDocker
+    }
+
+    String cromwellJarName = basename(cromwellJar)
+    String womtoolJarName = basename(womtoolJar)
+
+    String cromwellUploadUrl = "~{uploadUrl}?name=~{cromwellJarName}&label=~{cromwellJarName}"
+    String womtoolUploadUrl = "~{uploadUrl}?name=~{womtoolJarName}&label=~{womtoolJarName}"
+
+    String releaseRoot = "https://github.com/~{organization}/cromwell/releases/download/~{releaseVersion}/"
+
+    command <<<
+        # Do not use `set -x` or it will print the GitHub token!
+        set -euo pipefail
+
+        echo 'Upload the cromwell jar as an asset'
+        curl \
+            --fail --silent \
+            --header "Authorization: token $(cat ~{githubTokenFile})" \
+            --header "Content-Type: application/octet-stream" \
+            --data-binary @~{cromwellJar} \
+            "~{cromwellUploadUrl}"
+
+        echo 'Upload the womtool jar as an asset'
+        curl \
+            --fail --silent \
+            --header "Authorization: token $(cat ~{githubTokenFile})" \
+            --header "Content-Type: application/octet-stream" \
+            --data-binary @~{womtoolJar} \
+            "~{womtoolUploadUrl}"
+
+        echo 'Publish the draft'
+        curl \
+            --fail --silent \
+            --header "Authorization: token $(cat ~{githubTokenFile})" \
+            --request PATCH \
+            --data '{"draft": false, "tag": "~{releaseVersion}"}' \
```

Comment:
draft false?

Reply:
Yep, unless I made a typo, `draft: false` should flip the release from draft to published status.